.github/workflows/spell-check.yml

```
@@ -14,29 +13,32 @@ jobs:
   # This workflow contains a single job called "spell check"
   spell-check:
     runs-on: ubuntu-latest
-    container:
-      image: rocker/tidyverse:4.3.2
 
```

Do you want to add...

```suggestion
concurrency:
  # only one run per branch at a time
  group: ${{ github.workflow }}-${{ github.ref }}
  cancel-in-progress: true

```

---

The spellcheck is fast enough that I don't expect this to be an issue. Getting notifications of cancelled jobs is more annoying than the extra seconds of processing, I think.